test/documentation/.vercel/install.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+set -x
+set -e
+
+DOXYGEN_VERSION=1.9.1
```

Comment:
Doesn't have to be now, but I am wondering if there could be a way to generalize this...
We are setting the doxygen version in 2 github action workflows and now here...

Reply:
Yep, I think we should parse the content from github actions.